addons/point_of_sale/models/pos_order.py

```diff
@@ -649,7 +649,6 @@ def action_receipt_to_customer(self, name, client, ticket, order_ids=False):
                 'name': filename,
                 'type': 'binary',
                 'datas': base64.b64encode(report[0]),
-                'datas_fname': filename,
                 'store_fname': filename,
                 'res_model': 'account.move',
```

Comment:
@pimodoo Before you approve/disapprove, I also need to get your attention on this 'res_model' field. We are setting its value to 'account.move' but we are giving 'res_id' a value that comes from 'order_ids'. I think this is inconsistent.

This field's value should be 'pos.order' if we are making the attachment for pos.order record. Or if this attachment is for the invoice, then the value of 'res_id' should be 'orders[0].account_move.id'.

What do you think?